sources/assets/Stride.Core.Packages/NugetStore.cs

```diff
@@ -155,7 +155,29 @@ public static void UpdatePackageSource(ISettings settings, string name, string u
         /// <summary>
         /// Package Id of the Visual Studio Integration plugin.
         /// </summary>
-        public string VsixPluginId { get; } = "Stride.VisualStudio.Package";
+        public string VsixPackageId { get; } = "Stride.VisualStudio.Package";
+
+        /// <summary>
+        /// The different supported versions of Visual Studio
+        /// </summary>
+        public enum VsixSupportedVsVersion
+        {
+            VS2019,
+            VS2022
+        }
+
+        /// <summary>
+        /// A mapping of the supported versions of VS to a Stride release version range.  
+        /// For each supported VS release, the first Version represents the included earliest Stride version eligible for the VSIX and the second Version is the excluded upper bound.
+        /// </summary>
+        public IReadOnlyDictionary<VsixSupportedVsVersion, (PackageVersion, PackageVersion)> VsixVersionToStrideRelease { get; } = new Dictionary<VsixSupportedVsVersion, (PackageVersion, PackageVersion)>
```

Comment:
You should be able to define names for the parameters of those tuples by defining the tuple like so 
```cs
public IReadOnlyDictionary<VsixSupportedVsVersion, (min:PackageVersion, max:PackageVersion)>
```
You can then use those inside of `sources/launcher/Stride.Launcher/ViewModels/VsixVersionViewModel.cs` instead of `Item1` and `Item2`

Reply:
Great, thanks for the tip.  Will update.

Reply:
Just a note that the syntax was slightly different than what you mentioned but it works well and makes the code much more satisfying.